src/otx/core/model/visual_prompting.py

```diff
@@ -923,7 +926,7 @@ def _customize_inputs(  # type: ignore[override]
             if self.training:
                 points: list[Prompt] = []
                 bboxes: list[Prompt] = []
-                for prompt, label in zip(prompts, labels):  # type: ignore[arg-type]
+                for prompt, label in zip(prompts, labels["prompts"]):  # type: ignore[arg-type]
```

Comment:
Are polygons are supported by OTX OV model? MAPI wrapper already supports it, so we can just collect & forward the polygons here.

Reply:
I updated polygon support at L938-940 to use MAPI wrapper's polygon component.